src/pages/CustomRichTextEditorLink.tsx

```diff
@@ -0,0 +1,189 @@
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Button, ButtonVariants } from '@/ui/Button';
+import { FormElement } from '@/ui/FormElement';
+import { Inline } from '@/ui/Inline';
+import { Input } from '@/ui/Input';
+import { Stack } from '@/ui/Stack';
+
+/**
+ * Source: https://github.com/jpuri/react-draft-wysiwyg/blob/master/src/components/Option/index.js#L8
+ */
+function Option({
```

Comment:
Could you use arrow functions for codebase consistency? Unless y'all have changed the convention since I last worked on it 😄